tests/ballerina-integration-test/src/test/resources/websub/advanced_services/01_websub_publisher.bal

```diff
@@ -70,23 +71,29 @@ service publisher on publisherServiceEP {
     }
 
     @http:ResourceConfig {
-        methods: ["POST"]
+        methods: ["POST"],
+        path: "/notify/{subscriber}"
     }
-    resource function notify(http:Caller caller, http:Request req) {
+    resource function notify(http:Caller caller, http:Request req, string subscriber) {
         var payload = req.getJsonPayload();
         if (payload is error) {
             panic payload;
         }
 
-        http:Response response = new;
-        var err = caller->accepted(message = response);
+        if (subscriber == "8383") {
+            checkSubscriberAvailability(WEBSUB_PERSISTENCE_TOPIC_ONE, "http://localhost:8383/websub");
+        } else if (subscriber == "8484") {
+            checkSubscriberAvailability(WEBSUB_PERSISTENCE_TOPIC_ONE, "http://localhost:8484/websub");
```

Comment:
Are there scenarios where we ignore the `subscriber` value? Can't we remove the if-else blocks and say 
```suggestion
            checkSubscriberAvailability(WEBSUB_PERSISTENCE_TOPIC_ONE, "http://localhost:" + subscriber +  "/websub");
```